src/Element/NodeElement.php

```diff
@@ -237,6 +237,24 @@ public function selectOption($option, $multiple = false)
         $this->getDriver()->selectOption($this->getXpath(), $opt->getValue(), $multiple);
     }
 
+    /**
```

Comment:
There are multiple CS issues in the code, like missing spaces.

Reply:
I fixed the missing spaces before and after the equality signs and the logical expressions.